.github/workflows/prerelease.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.9.7'
+  GO_VERSION: '1.16'
```

Comment:
Why did we change version of go to 1.16? Did we do testing of this change for all snmp features for all supported platforms?

Reply:
We are taking the chance of updating the SDK version to 3.6.7 to update the repositories to Go Modules and update the go version, I adde an integration test where I test this integration, I executed it in the old and new version of the repo and it worked as expected, but sure there is not a full test with all the platforms. How do you think I should proceed to make a full test? How do you normally do it?

Reply:
I guess I can check the results of executing different Vagrant boxes with the agent and integration after and before the update reading from an snmp host to ensure all the data is the same even if it's covered in the integration test, one thing that surprised me about this integration is that it doesn't have any unit test. I actually found an error thanks to the linter that would have been covered in a unit test .